examples/graphiql-2-rfc-context/src/components/GraphiQL.tsx

```diff
@@ -361,7 +361,7 @@ class GraphiQLInternals extends React.Component<
             console: variables,
           }}
           navPanels={[
-            // TODO: rewrite this for plugin API
+            // eslint-disable-next-line no-constant-condition -- TODO: rewrite this for plugin API
```

Comment:
it's better to use `eslint-disable` directive that turns rule as warning for this particular case